.github/workflows/windows-ci.yml

```diff
@@ -0,0 +1,263 @@
+        rendering_mode: [legacy, drawable]
+        exclude:
+          - renderer: egl
+            rendering_mode: drawable
+          - renderer: vulkan
+            rendering_mode: legacy
+    runs-on: windows-2022
+    steps:
+      - run: |
+          git config --system core.longpaths true
+
+      - uses: actions/checkout@v4
+        with:
+          submodules: recursive
+          fetch-depth: 0
+
+      - name: Initialize CodeQL
+        uses: github/codeql-action/init@v3
+        with:
+          languages: cpp
+
+      - if: matrix.rendering_mode == 'legacy'
+        shell: bash
+        run: echo rendering_mode_flag_cmake=-DMLN_LEGACY_RENDERER=ON >> "$GITHUB_ENV"
+
+      - if: matrix.rendering_mode == 'drawable'
+        shell: bash
+        run: echo rendering_mode_flag_cmake=-DMLN_DRAWABLE_RENDERER=ON >> "$GITHUB_ENV"
+
+      - if: matrix.renderer == 'opengl'
+        shell: bash
+        run: echo renderer_flag_cmake="-DMLN_WITH_OPENGL=ON" >> "$GITHUB_ENV"
+
+      - if: matrix.renderer == 'egl'
+        shell: bash
+        run: echo renderer_flag_cmake="-DMLN_WITH_EGL=ON" >> "$GITHUB_ENV"
+
+      - if: matrix.renderer == 'vulkan'
+        shell: bash
+        run: echo renderer_flag_cmake="-DMLN_WITH_VULKAN=ON -DMLN_WITH_OPENGL=OFF" >> "$GITHUB_ENV"
+
+      - if: matrix.renderer == 'osmesa'
+        shell: bash
+        run: echo renderer_flag_cmake="-DMLN_WITH_OSMESA=ON" >> "$GITHUB_ENV"
+
+      - uses: ilammy/msvc-dev-cmd@v1
+
+      - name: Export GitHub Actions cache environment variables
+        uses: actions/github-script@v7
+        with:
+          script: |
+            core.exportVariable('ACTIONS_CACHE_URL', process.env.ACTIONS_CACHE_URL || '');
+            core.exportVariable('ACTIONS_RUNTIME_TOKEN', process.env.ACTIONS_RUNTIME_TOKEN || '');
+
+      - uses: mozilla-actions/sccache-action@v0.0.6
+
+      - name: Initialize sccache
+        run: |
+          sccache --start-server
+          sccache --zero-stats
+
+      - name: Configure MapLibre Native Core
+        env:
+          CI: 1
+          VCPKG_BINARY_SOURCES: "clear;x-gha,readwrite"
+          VCPKG_KEEP_ENV_VARS: CMAKE_CXX_COMPILER_LAUNCHER;CMAKE_C_COMPILER_LAUNCHER
+          CMAKE_C_COMPILER_LAUNCHER: sccache
+          CMAKE_CXX_COMPILER_LAUNCHER: sccache
+        run: |
+          cmake --version
+          cmake -B build -GNinja `
+            -DCMAKE_POLICY_DEFAULT_CMP0141=NEW `
+            -DCMAKE_MSVC_DEBUG_INFORMATION_FORMAT=Embedded `
+            -DCMAKE_BUILD_TYPE=RelWithDebInfo `
+            ${{ env.renderer_flag_cmake }} `
+            ${{ env.rendering_mode_flag_cmake }}
+
+      - name: Build MapLibre Native Core
+        run: |
+          cmake --build build --target mbgl-core mbgl-test-runner mbgl-render-test-runner mbgl-expression-test mbgl-render mbgl-benchmark-runner
+
+      # mbgl-render (used for size test) & mbgl-benchmark-runner
+
+      - name: Upload mbgl-render as artifact
```

Comment:
These two steps can be removed, since we don't do a size test or benchmark in a subsequent workflow like on Linux.

Reply:
Removed!